schema/ddl/20240312173400_create_sale.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS sales (
+    id char(26) NOT NULL PRIMARY KEY,
+    barcode_id char(26) NOT NULL,
+    account_id char(26) NOT NULL,
+    purchase_date TIMESTAMP NOT NULL,
```

Comment:
[imo] こちらも同様に，下記の方が良きです！
```suggestion
    purchased_at timestamptz NOT NULL,
```